code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -323,7 +328,7 @@
 	our_air.react(src)
 
 	update_visuals()
-	if(!consider_superconductivity(starting = TRUE) && !active_hotspot) //Might need to include the return of react() here
+	if(!consider_superconductivity(TRUE) && !active_hotspot) //Might need to include the return of react() here
```

Comment:
This is a decent hit to readability, a profile of just this would be appreciated